civo/instances/resource_instance.go

```diff
@@ -278,19 +279,30 @@ func resourceInstanceCreate(ctx context.Context, d *schema.ResourceData, m inter
 	config.Tags = tags
 
 	log.Printf("[INFO] creating the instance %s", d.Get("hostname").(string))
-	err := utils.RetryUntilSuccessOrTimeout(func() error {
-		instance, err := apiClient.CreateInstance(config)
-		if err != nil {
-			return err
-		}
-		d.SetId(instance.ID)
-		return nil
-	}, 10*time.Second, 2*time.Minute)
 
+	instance, err := apiClient.CreateInstance(config)
 	if err != nil {
-		return diag.Errorf("[ERR] failed to create instance after multiple attempts: %s", err)
+		if errors.Is(err, civogo.DatabaseInstanceDuplicateNameError) {
```

Comment:
Is this the only error we want to verify? There are multiple errors are present, What will happen to those?